### PR TITLE
Add bio profile

### PR DESCRIPTION
**Pull request by maitrungdong**
This is just some line about my bio.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # hello-world
 This is to use to demo Github
+Hi everyone.
+I am Mai Trung Dong. I am a student in HCMUS.
+My expertise's information technology.
+I spend the time of two years to learning some basic subject in the college.
+So now, I am a third-year student. I feel quite bored about some things in school. For example,
+my faculty is very very "it gai - dong luc cua dan ong moi khi den truong".
```
Comment (maitrucloan) [inline README.md:8]: Tao lao het suc vay Mai Trung Dong.

**Comment by maitrucloan** [comment]
Ua vay thi bio of toi dau ha **ban Kia.**

File: README.md (hunk 1 above)
Comment (maitrungdong) [reply README.md:8]: Ak cai nay la do toi ngu tieng Anh nen khoong biet viet nhu the nao

**Comment by maitrungdong** [comment]
> Ua vay thi bio of toi dau ha **ban Kia.**

**Comment by maitrungdong** [comment]
> > Ua vay thi bio of toi dau ha **ban Kia.**

Toi xin loi, vi ban moi vao nen chua co

**Pull request merged**